plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -665,6 +665,24 @@ public void testInsertIntoNotNullColumn()
         }
     }
 
+    @Test
+    @Override
+    public void testInsertMap()
+    {
+        // TODO: Add more types here
+        testMapRoundTrip("INTEGER", "2");
+        testMapRoundTrip("VARCHAR", "CAST('foobar' AS VARCHAR)");
+    }
+
+    private void testMapRoundTrip(String valueType, String value)
+    {
+        try (TestTable table = newTrinoTable("test_insert_map_", "(col map(INTEGER, %s) NOT NULL)".formatted(valueType))) {
```

Comment:
This fails without the `NOT NULL` right now, gotta figure that out or at least improve the error message